pages/docs/concepts/server.md

```diff
@@ -0,0 +1,21 @@
+
+## What is a server?
+A server is a representation of a messaging broker system where connections and communication between a producer and a consumer is established. Unlike traditional API servers which are dependent on request/response, message broker interactions occur back and forth over different channels.
+
+## What is the purpose of servers?
+Servers play an important role as they maintain a relationship between producers and consumers. When designing and setting up an event-driven application, servers are in charge of delivering asynchronous messages from the producer to the consumers through the use of channels. With the integration of different messaging protocols, servers can transmit and exchange messages between clients.
+
+Server diagram made by mermaid.js :
+
+```mermaid
+flowchart LR
+    A[producer]
+    A --> a1[channel1]
+    A --> a2[channel2]
+    subgraph one[server/broker]
```

Comment:
```suggestion
    subgraph one[broker]
```

we can say just `broker` I think, and just describe what the diagram shows, that it shows: In broker-centric architecture, broker is considered to be a server, even thought of course technically producer and consumer can also be a server. But the point is to explain from AsyncAPI perspective. For the above diagram, you would create 3 AsyncAPI documents, one for `producer`, one for `consumer1`, and one for `consumer2`. The `server` information that you would provide in these AsyncAPI files, would point to the `broker` itself. In other words, in AsyncAPI file for `producer` you point to the `broker` as the server, not the `producer` itself

but we fundamentally need to agree if in concept docs we want to just talk about `Server` concept only in general, or maybe always provide AsyncAPI context 🤔 

btw, love the way you did it with mermaid, I wonder what you think about it 😄

Reply:
I know the goal was to define the server concept only, but if there's a need to also explain the diagram I think we should do it. The more information we provide the better people understand the concept. @alequetzalli any thoughts on this?

Reply:
Yes absolutely @thulieblack, I completely agree. I would add a written explanation of your diagram and we will add it right underneath it in the markdown. 😀👍🏽👍🏽 

Do you feel comfortable writing that explanation or would you like more support? ❤️

Reply:
Lukasz has already provided some context, so I'm confident in adding the explanation😊